### ISSUE TITLE
Add option to delete recurring events

### ISSUE THREAD

**Issue by DOFfactory** [marcoancona/MineTime#247]
Currently if one wants to delete a recurring event, the options are:

- Delete this event
- Delete all events

It would be great functionality to introduce to "Delete _from_ this event onwards" and "Delete all previous events" from the selected one, as often this is important for archiving/planning purposes.

**Comment by marcoancona** [comment]
Right, this will come with the next major update. Thanks for reporting.
